src/server.js

```diff
@@ -0,0 +1,10 @@
+require('dotenv').config();
+const express = require('express');
+const db = require('../database/config');
```

Comment:
since you are not reusing db, no need to declare a varaible, just require database config file